step-capstone/src/components/TimeLine.js

```diff
@@ -0,0 +1,167 @@
+          <ArrowLeft 
+            displayDate = {props.displayDate}
+            onChangeDisplayDate = {props.onChangeDisplayDate}
+          />
+          <button className="secondary">EST</button>
+          <PickDisplayDate 
+            displayDate = {props.displayDate}
+            onChangeDisplayDate = {props.onChangeDisplayDate}
+          />
+          <ArrowRight 
+            displayDate = {props.displayDate}
+            onChangeDisplayDate = {props.onChangeDisplayDate}
+          />
+        </div> 
+        <div className="gap"></div>
+      </header>
+
+      <table>
+        <thead className="header name">
+          <tr>
+            <th className="headcol"></th>
+            <th>Finalized</th>
+          </tr>
+        </thead>
+      </table>
+    </div>
+  )
+}
+
+function handleOnClickInterval(idV) {
+  console.log(idV);
+}
+
+function OneHourInterval(props) {
+  return (
+    <div className="OneHourInterval">
+      <tr className="OneHourInterval">
+          <td className="headcol">{props.idV + ":00"}</td>
+          <td className="Interval" id={props.idV +":00"} onClick={() => handleOnClickInterval(props.idV + ":00")}></td>
+      </tr>
+      <tr className="OneHourInterval">
+        <td className="headcol"></td>
+        <td className="Interval" id={props.idV + ":30"} onClick={() => handleOnClickInterval(props.idV + ":30")}></td>
+      </tr>
+    </div>
+  )
+}
+export default function TimeLine(props) {
+  return (
+    <div className="overall">
+        <div className="calendar">
+          <Header 
+            displayDate = {props.displayDate}
+            onChangeDisplayDate = {props.onChangeDisplayDate}  
+          />
+
+          <div className="outer">
+            <div className="wrap"> 
+              <table className="offset">
+                <tbody>
+                  <tr></tr>
+                  <OneHourInterval idV="0" />
+                  <OneHourInterval idV="1" />
+                  <OneHourInterval idV="2" />
+                  <OneHourInterval idV="3" />
+                  <OneHourInterval idV="4" />
+                  <OneHourInterval idV="5" />
+                  <OneHourInterval idV="6" />
+                  <OneHourInterval idV="7" />
+                  <OneHourInterval idV="8" />
+                  <OneHourInterval idV="9" />
+                  <OneHourInterval idV="10" />
+                  <OneHourInterval idV="11" />
+                  <OneHourInterval idV="12" />
+                  <OneHourInterval idV="13" />
+                  <OneHourInterval idV="14" />
+                  <OneHourInterval idV="15" />
+                  <OneHourInterval idV="16" />
+                  <OneHourInterval idV="17" />
+                  <OneHourInterval idV="18" />
+                  <OneHourInterval idV="19" />
+                  <OneHourInterval idV="20" />
+                  <OneHourInterval idV="21" />
+                  <OneHourInterval idV="22" />
+                  <OneHourInterval idV="23" />
```

Comment:
Not being a front end person I wonder if this could be generated, but you're the expert now so up to you.

Reply:
That's a good idea. I will put a for loop in to generate this.

Reply:
Addressed